### ISSUE TITLE
Introduce DB for configuration of Things, Placements and so on

### ISSUE THREAD

**Issue by s-kostyuk** [s-kostyuk/everpl#30]
Introduce a database for storing of configurations of all Things in the system. Replace current config-file configurations of Things with DB alternatives

So, the subtasks:
- [x] create a set of classes that will store configuration of Things and Connections:
  - [x] ThingSettings
  - [x] ConnectionSettings
- [x] declare abstract repositories:
  - [x] ThingSettingsRepository;
  - [x] ConnectionSettingsRepository;
  - [x] PlacementsRepository;
- [x] set up ORM mapping:
  - [x] ThingSettings table;
  - [x] ConnectionSettings table;
  - [x] Placements table;
- [x] implement DB-specific repositories:
  - [x] ThingSettingsRepository;
  - [x] ConnectionSettingsRepository;
  - [x] PlacementsRepository;
- [ ] set up Services to use corresponding repos.

**Comment by s-kostyuk** [comment]
Changed title.

Actually, at least three tables in the DB is needed:
- a table with configuration for Things;
- a table with configuration for Connections;
- a table with data about Placements.

And, with some reservations, SQL databases fit good to store this data. Considering this and an absence of mature embedded document-oriented databases, SQLite will be used for this task.

New status: In progress

**Comment by s-kostyuk** [comment]
Blocked by #32

**Comment by s-kostyuk** [comment]
OK, the last subtask ("set up Services to use corresponding repos") will not be implemented in version v0.2 and is postponed to the next version of everpl. The reason for that is a need to review Thing and Connection process for one more time before the final implementation will be created
